modules/dynamic-plugin-proxy-service.adoc

```diff
@@ -0,0 +1,43 @@
+// Module included in the following assemblies:
+//
+// * web_console/dynamic-plugin/deploy-plugin-cluster.adoc
+
+:_mod-docs-content-type: CONCEPT
+[id="dynamic-plugin-proxy-service_{context}"]
+= Plugin service proxy
+
+You can declare a service proxy in its `ConsolePlugin` resource by using the `spec.proxy` array field if you need to make HTTP requests to an in-cluster service with your plugin. The console backend exposes the `/api/proxy/plugin/<plugin-name>/<proxy-alias>/<request-path>++?++<optional-query-parameters>` endpoint to proxy the communication between plugin and the service. A proxied request uses a _service CA bundle_ by default. The service must use HTTPS.
```

Comment:
Maybe add a quick explanation of why you'd want to do this somewhere (i.e., if your plugin needs to make a request to a service running on the cluster not exposed through a route).